about/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Первая страница</title>
+    <font size = "2" </font>
+<link rel="stylesheet" href="/about/style.css"/>
```

Comment:
nice try, but the current href doesn't work. This html file is located on the same folder with style.css, to connect you need to specify just "style.css"